test/end-to-end.js

```diff
@@ -562,6 +552,45 @@ describe('end-to-end', function() {
       expect(config).to.have.property('legacyExplorer', false);
       done();
     });
+
+    it('enables AccessToken invalidation', function(done) {
+      bootSandboxWithOptions({}, function(err, app) {
+        if (err) return done(err);
+
+        // enable the authentication
+        app.dataSource('db', { connector: 'memory' });
+        app.enableAuth({ dataSource: 'db' });
+
+        // create and login a user
+        var CREDENTIALS = { email: 'test@example.com', password: 'pass' };
+        var User = app.models.User;
+        var user;
+
+        async.series([
+          function createUser(next) {
+            User.create(CREDENTIALS, function(err, u) {
+              if (err) return done(err);
+              user = u;
+              next();
+            });
+          },
+          function createToken(next) {
+            User.login(CREDENTIALS, next);
+          },
+          function changeEmail(next) {
+            user.updateAttribute('email', 'new@example.com', next);
+          },
+          function verify(next) {
+            app.models.AccessToken.find(function(err, list) {
+              if (err) return next(err);
+              list = list.map(function(t) { return t.toObject(); });
+              expect(list).to.eql([]);
```

Comment:
https://github.com/moll/js-must#beware-of-libraries-that-assert-on-property-access

I could add `dirty-chai` and use `.to.be.empty()`, but I feel it's not worth the effort now.